examples/systemd/post-install

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# this post upgrade script is run each time the teleport package is upgraded
+
+set -eu
+
+case "$1" in
+    configure)
+        # Define old paths and new base directory.
+        OLD_PATHS=(
+            "/usr/local/bin/teleport"
+            "/usr/local/bin/tsh"
+            "/usr/local/bin/tctl"
+            "/usr/local/bin/tbot"
+        )
+        NEW_BASE_DIR="/usr/local/teleport-system/bin"
+
+        echo "Cleaning up old files and creating symlinks where necessary..."
+
+        for FILE in "${OLD_PATHS[@]}"; do
+            BASENAME=$(basename "$FILE")
+            NEW_PATH="$NEW_BASE_DIR/$BASENAME"
+
+            if [ -e "$FILE" ] && [ ! -L "$FILE" ]; then
+                echo "Removing old binary at $FILE..."
+                rm -f "$FILE"
+            fi
+
+            if [ ! -e "$FILE" ] && [ ! -L "$FILE" ]; then
+                echo "Creating symlink at $FILE pointing to $NEW_PATH..."
+                ln -s "$NEW_PATH" "$FILE"
+            fi
+        done
+
+        echo "Post-upgrade cleanup and symlink creation completed."
+        ;;
+    remove|deconfigure)
```

Comment:
Should we run this on `deconfigure`, or just `remove`? Seems like we'd want these symlinks to stay around until the package is fully removed.

Reply:
this is actually `prerm` script part, for local testing for me was easier if it is one file. We need to call `teleport-update unlink` before binaries from package are removed if we want to have `unlink` command. Deconfigure should be call in case of package downgrade as far as understood correctly the documentation